sh/test.sh

```diff
@@ -0,0 +1,33 @@
+# Automated Test for CFRK
+
+# NOTA: Linux nega a permissão ao rodar esse script!
+#       Executar este comando antes de rodar o bash:
+#
+#          sudo chmod u+x ./test.sh
+#
+#!/usr/bin/env bash
+
+echo "Automated test for CFRK!"
+cd ../bin
+
+# ./cfrk seq1.fasta out.cfrk 2 12 8192
+
+diff out.cfrk out-seq1.cfrk
+ret=$?
+
+if [[ $ret -eq 0 ]]; then
```

Comment:
Esse `if `pode ser evitado (se o objetivo dele for somente o CI) adicionando a opção -e no shebang:  `#!/bin/bash -e`
Assim, automaticamente o script se encerra com um erro de execução.

Reply:
Boa dica! rsrs Vou adicionar o -e.